tutorials/034 - Distributed Calls on Remote Cluster.ipynb

```diff
@@ -0,0 +1,186 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![AWS SDK for pandas](_static/logo.png \"AWS SDK for pandas\")](https://github.com/aws/aws-sdk-pandas)\n",
+    "\n",
+    "# 7 - Distributed Calls on Remote Cluster\n",
+    "\n",
+    "[SDK for pandas](https://github.com/aws/aws-sdk-pandas) supports distribution of specific calls using [ray](https://www.ray.io/)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "pip install awswrangler[\"distributed\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configure and Build Ray Cluster in AWS.\n",
+    "\n",
+    "#### Build Prerequisite Infrastructure\n",
+    "\n",
+    "Build a security group and IAM instance profile for the Ray Cluster to use.\n",
+    "\n",
+    "[<img src=\"https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png\">](https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=RayPrerequisiteInfra&templateURL=https://aws-data-wrangler-public-artifacts.s3.amazonaws.com/cloudformation/ray-prerequisite-infra.json)\n",
+    "\n",
+    "#### Configure Ray Cluster Configuration\n",
+    "Start with a cluster configuration file (YAML)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "touch config.yml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Replace all values to match your desired region, account number and name of resources deployed by the above CloudFormation Stack.\n",
+    "\n",
+    "[Click here](https://console.aws.amazon.com/ec2/home?region=us-east-1#Images:visibility=public-images;search=:ray-amzn-wheels_latest_amzn_ray-1.9.2-cp38;v=3;$case=tags:false%5C,client:false;$regex=tags:false%5C,client:false) to find the Ray AMI for your desired region. The example configuration below uses the AMI for `us-east-1`"
```

Comment:
So ray aws config provider default to an image with Python 3.7 which we are not supporting at the moment in our >3.0.0 releases. That's why I have this part in there.